vigranumpy/src/core/colors.cxx

```diff
@@ -382,7 +382,7 @@ void pythonGray2QImage_ARGB32Premultiplied(
     TmpType pixelF;
     
     TmpType normalizeLow, normalizeHigh; 
-    if(normalize != boost::python::object())
+    if(normalize.pyObject() != boost::python::object().ptr())
```

Comment:
Thanks a lot!
I think I would have compared with ``Py_None``, wouldn't you prefer that?
(Not only that it's shorter, but it also does not create a temporary.) Anyhow, would be safe to merge.